### PR TITLE
Update creating-3d-models-for-use-in-the-windows-mixed-reality-home.md

### PR DESCRIPTION
**Pull request by thmignon**

### CODE REVIEW

File: mixed-reality-docs/creating-3d-models-for-use-in-the-windows-mixed-reality-home.md [hunk 1]
```diff
@@ -261,6 +262,8 @@ The following material properties are ignored from core spec:
 
 Windows MR does not support primitive mode lines and points. 
 
+Only a single UV vertex attribute is supported
```
Comment (najadojo) [inline mixed-reality-docs/creating-3d-models-for-use-in-the-windows-mixed-reality-home.md]: Missing period

Comment (thmignon) [reply mixed-reality-docs/creating-3d-models-for-use-in-the-windows-mixed-reality-home.md]: Good catch, added in latest commit

**Comment by mattzmsft** [comment]
@thmignon by April 1803 update do you mean the Windows 10 April 2018 Update? If so, we need to be consistent about using the public terminology for these releases (and link them to the current release notes article, release-notes.md)

**Comment by thmignon** [comment]
@mattzmsft  yea I got the prerelease name mixed up. I've updated it with a hyperlink to the support doc on updating.

**Pull request merged**